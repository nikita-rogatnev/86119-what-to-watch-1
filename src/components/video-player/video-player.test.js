import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import VideoPlayer from "./video-player";

import mockData from "../../mocks/mock-data";

Enzyme.configure({adapter: new Adapter()});

it(`VideoPlayer renders correctly`, () => {
  const tree = shallow(<VideoPlayer
    previewImage={mockData[0].previewImage}
    previewVideoLink={mockData[0].previewVideoLink}
  />);

  expect(toJson(tree)).toMatchSnapshot();
});

