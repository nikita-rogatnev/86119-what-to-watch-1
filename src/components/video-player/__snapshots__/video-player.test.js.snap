// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer renders correctly 1`] = `
<Route>
  <Component />
</Route>
`;
